pkgs/os-specific/linux/iotop-c/default.nix

```diff
@@ -0,0 +1,28 @@
+{stdenv, fetchFromGitHub, lib, ncurses, pkg-config }:
+
+stdenv.mkDerivation rec {
+  pname = "iotop-c";
+  version = "v1.17";
+
+  src = fetchFromGitHub {
+    owner = "Tomas-M";
+    repo = "iotop";
+    rev = version;
+    sha256 = "0hjy30155c3nijx3jgyn5kpj293632p0j6f3lf5acdfax1ynav86";
+  };
+
+  buildInputs = [ ncurses pkg-config ];
+  buildPhase = "make DESTDIR=$out";
+  installPhase = ''
+    make install DESTDIR=$out
+    ln -s $out/usr/sbin $out/bin
+    mv $out/usr/sbin/iotop $out/usr/sbin/${pname}
+  '';
+
+  meta = with lib; {
+    description = "iotop identifies processes that use high amount of input/output requests on your machine";
+    homepage = "https://github.com/Tomas-M/iotop";
+    license = licenses.gpl2;
```

Comment:
Missing maintainer and is the license gpl2Only or gpl2Plus?

Please add yourself as a maintainer by adding an alphabetically sorted entry to `maintainers/maintainer-list.nix` and commit this change into a separate commit with the message ``maintainers: add NAME``.